app/controllers/projects/extracts_controller.rb

```diff
@@ -10,11 +10,14 @@ def show
       redirect_to @project, notice: msg
       return
     end
+
+    @value_set = Dataset::ValueSet.new
```

Comment:
> and shows form errors inline.

In what situation do these appear, or do you mean that we _could_ do that in future now?

Reply:
These show now when the format is incorrect but we're not using the numeric type yet as far as I can tell.